shared/db/collections/sentences-meta.js

```diff
@@ -240,6 +290,20 @@ export default class SentencesMeta {
     return result.data;
   }
 
+  async getAllPaginated(language) {
```

Comment:
This is essentially `getAllValidatedSentences` without the filter now, after both PRs?